Bio/Phylo/TreeConstruction.py

```diff
@@ -545,43 +545,42 @@ class DistanceTreeConstructor(TreeConstructor):
 
     Examples
     --------
-    >>> from TreeConstruction import DistanceTreeConstructor
+    >>> from Bio.Phylo.TreeConstruction import DistanceTreeConstructor
     >>> constructor = DistanceTreeConstructor()
 
     UPGMA Tree:
 
     >>> from Bio.Phylo.TreeConstruction import DistanceCalculator
     >>> from Bio import AlignIO
-    >>> aln = AlignIO.read(open('Tests/TreeConstruction/msa.phy'), 'phylip')
+    >>> aln = AlignIO.read(open('./TreeConstruction/msa.phy'), 'phylip')
     >>> calculator = DistanceCalculator('identity')
     >>> dm = calculator.get_distance(aln)
     >>> upgmatree = constructor.upgma(dm)
     >>> print(upgmatree)
     Tree(rooted=True)
-        Clade(name='Inner4')
-            Clade(branch_length=0.171955155115, name='Inner1')
-                Clade(branch_length=0.111111111111, name='Epsilon')
-                Clade(branch_length=0.111111111111, name='Delta')
-            Clade(branch_length=0.0673103855608, name='Inner3')
-                Clade(branch_length=0.0907558806655, name='Inner2')
-                    Clade(branch_length=0.125, name='Gamma')
-                    Clade(branch_length=0.125, name='Beta')
-                Clade(branch_length=0.215755880666, name='Alpha')
+        Clade(branch_length=0, name='Inner4')
+            Clade(branch_length=0.18749999999999994, name='Inner1')
+                Clade(branch_length=0.07692307692307693, name='Epsilon')
+                Clade(branch_length=0.07692307692307693, name='Delta')
+            Clade(branch_length=0.11057692307692304, name='Inner3')
+                Clade(branch_length=0.038461538461538464, name='Inner2')
+                    Clade(branch_length=0.11538461538461536, name='Gamma')
+                    Clade(branch_length=0.11538461538461536, name='Beta')
+                Clade(branch_length=0.15384615384615383, name='Alpha')
```

Comment:
I am slightly concerned that the numbers have changed - was there a major change to the upgma code recently?

Reply:
There has not been changes on upgma, or changes for a while, maybe the order of the alignments on msa.phy might have affected this.

Reply:
Nope the order doesn't affect the values.

Reply:
Very strange. A little mystery there, but another factor making me less comfortable in using this as a docstring test requiring perfect agreement.

Reply:
There may have been a subtle bugfix within the last couple years. In other words, I don't know that the original UPGMA implementation was perfect. If there are deep concerns this could be compared to the output of phylip's implementation.